config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ images:
 - name: quay.io/kata-containers/kata-deploy
   newName: quay.io/kata-containers/kata-deploy-ci
   # FIXME: Remove this once https://github.com/confidential-containers/operator/issues/391 is resolved
-  newTag: kata-containers-43dca8deb4891678f8a62c112749ac0938b373c6-amd64
+  newTag: kata-containers-1f6a8baaf187e38e9e6eae0f4a225f77cdefd3a6-amd64
```

Comment:
Hi @fidencio !

Something is wrong. Shouldn't it be `ghcr.io/kata-containers/kata-deploy-ci:10169-1f6a8baaf187e38e9e6eae0f4a225f77cdefd3a6-amd64` as published in https://github.com/kata-containers/kata-containers/actions/runs/10405169709/job/28815750111?pr=10169 ?

quay.io/kata-containers/kata-deploy-ci:kata-containers-1f6a8baaf187e38e9e6eae0f4a225f77cdefd3a6-amd64 doesn't exist

Reply:
Yeah, you're correct.